pkg/api/addons.go

```diff
@@ -287,6 +298,29 @@ func (cs *ContainerService) setAddonsConfig(isUpdate bool) {
 		},
 	}
 
+	defaultsAADPodIdentityAddonsConfig := KubernetesAddon{
```

Comment:
"Promoted" this to the "container addons" interface as it makes more sense for this addon to be user-configurable.